dist/js/bootstrap-switch.js

```diff
@@ -525,22 +525,38 @@
           _this4.setPrevOptions();
           _this4._width();
           _this4._containerPosition();
+
```

Comment:
can you avoid making changes to dist/ files when making PRs, as this makes all PRs conflict.